src/Raven.Studio/gulpfile.js

```diff
@@ -234,7 +234,7 @@ gulp.task('z_mochaTests', function () {
 });
 
 gulp.task('test', [ 'z_compile:test' ], function (cb) {
-    return runSequence('z_generate-test-list', 'z_mochaTests', cb);
+    return runSequence('z_generate-test-list', /*'z_mochaTests', */cb);
```

Comment:
remove or put TODO ?

Reply:
For now I do not want to run the tests as part of the watch, but the test method should run them probably. So I commented this until later.

Reply:
it is 'test' task, not 'watch' so maybe watch task is broken. Watch indeed should only compile files but not running tests. Could you fix that then? When running 'test' task we want to run tests. Current change breaks this feature.

Reply:
Reverted, https://github.com/ravendb/ravendb/pull/3785